src/tactic/lint/type_classes.lean

```diff
@@ -300,6 +301,7 @@ attribute [nolint decidable_classical] dec_em not.decidable_imp_symm
 
 private meta def has_coe_to_fun_linter (d : declaration) : tactic (option string) :=
 retrieve $ do
+tt ← return d.is_trusted | pure none,
```

Comment:
I think I was thrown off by this line, which is not a doc change

Reply:
Ah, I forgot I also changed that. Should have put that in the commit message.